src/shared/components/notifications/Notifications.tsx

```diff
@@ -43,11 +44,12 @@ const Notifications: FC = () => {
               id={id}
               icon={icon}
               duration={duration}
-              size={ComponentSize.ExtraSmall}
+              size={ComponentSize.Small}
               gradient={gradient}
               onTimeout={handleDismiss}
               onDismiss={handleDismiss}
               testID={`notification-${style}`}
+              style={{maxWidth: '600px', alignItems: 'center'}}
```

Comment:
Not sure if we want to change notifications all over the app do we?

Reply:
This was requested by @mavarius

Reply:
K, @mavarius we cool to update all the notifications? If that's the case, set the styling for the `Notifications` in clockface so that's the default? If not, then should we make this a specific UI-default with a className setting these properties?